lib/measures/generic_qaqc/resources/check_eui_by_end_use.rb

```diff
@@ -38,33 +38,46 @@ module OsLib_QAQC
 
   # checks the number of unmet hours in the model
   def check_eui_by_end_use(category, target_standard, min_pass, max_pass, name_only = false)
+
     # summary of the check
     check_elems = OpenStudio::AttributeVector.new
     check_elems << OpenStudio::Attribute.new('name', 'End Use by Category')
     check_elems << OpenStudio::Attribute.new('category', category)
-    check_elems << OpenStudio::Attribute.new('description', "Check end use by category against #{target_standard} DOE prototype buildings.")
+
+    # update display sttandard
+    if target_standard.include?('90.1')
+      display_standard = "ASHRAE #{target_standard}"
+    else
+      display_standard = target_standard
+    end
 
     # stop here if only name is requested this is used to populate display name for arguments
     if name_only == true
       results = []
+      check_elems << OpenStudio::Attribute.new('description', "Check model consumption by end use against #{target_standard} DOE prototype building.")
       check_elems.each do |elem|
         results << elem.valueAsString
       end
       return results
     end
 
-    # Versions of OpenStudio greater than 2.4.0 use a modified version of
-    # openstudio-standards with different method calls.  These methods
-    # require a "Standard" object instead of the standard being passed into method calls.
-    # This Standard object is used throughout the QAQC check.
-    if OpenStudio::VersionString.new(OpenStudio.openStudioVersion) < OpenStudio::VersionString.new('2.4.3')
-      use_old_gem_code = true
-    else
-      use_old_gem_code = false
+    begin
+
+      # setup standard
       std = Standard.build(target_standard)
-    end
+      target_eui = std.model_find_target_eui(@model)
+
+      # gather building type for summary
+      bt_cz = std.model_get_building_climate_zone_and_building_type(@model)
```

Comment:
`model_get_building_climate_zone_and_building_type` no longer exists.  It's called `model_get_building_properties` now